### PR TITLE
Master bugfix

### PR DESCRIPTION
**Pull request by jdent02**
Fixes to layer file output and internal denoise

### CODE REVIEW

File: properties.py [hunk 1]
```diff
@@ -203,7 +203,7 @@ def update_name(self, context):
 
 aov_mapping = [ 
                  
-                 ("rgba", "Combined (rgba)", "Combined (rgba)"),
+                 ("Ci,a", "Combined (rgba)", "Combined (rgba)"),
```
Comment (bsavery) [inline properties.py]: So just a thought!  For main displays do we want rgba or Ci,a?

File: export.py [hunk 2]
```diff
@@ -2844,41 +2852,58 @@ def export_display(ri, rpass, scene):
...
-                        dspy_name = image_base + '.%s.%s.' % (layer_name, aov_name) + ext
-                        ri.Display('+' + dspy_name, display_driver, aov.channel_name, params)
+                        dspy_name = image_base + \
+                            '.%s.%s.' % (layer_name, aov_name) + ext
+                        ri.Display('+' + dspy_name, display_driver,
+                                   aov.channel_name, params)
                         rpass.output_files.append(dspy_name)
 
-    if rm.do_denoise and not rpass.is_interactive:
+    if rm.do_denoise or rm.external_denoise and not rpass.is_interactive:
```
Comment (bsavery) [inline export.py]: You could have rm.external_denoise checked and not be doing an external render..  in this case denoise data would be exported.

File: properties.py (hunk 1 above)
Comment (jdent02) [inline properties.py]: I used that as a shortcut for multilayer export.  Reviewing my code 'rgba' would work just as well.

File: export.py (hunk 2 above)
Comment (jdent02) [inline export.py]: True.  Incidentally the same thing happens if denoise is checked in the internal render options and an external render is done.  I'll add a check that will output the denoise channels only if the corresponding denoise option is checked for internal or external rendering.

**Comment by jdent02** [comment]
Fixed....

**Pull request merged**